msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -10,15 +10,55 @@ ms.author: v-npaladugu
 
 A configuration page is a special type of [content page](content-page.md). The users configure some aspects of the Microsoft Teams app using the configuration page and use that configuration as part of the following:
 
-* A channel or group chat tab: Collect information from the users and set the `contentUrl` of the content page to be displayed.
+* A personal, channel, or group chat tab: Collect information from the users and set the `contentUrl` of the content page to be displayed.
```

Comment:
This is incorrect. You can't have a configuration dialog or experience for _personal_ (personally-scoped) static tabs. And since configurableTabs don't work in the personal scope, there's no option for a user to configure anything in the 'personal' scope.

Reply:
Updated